### PR TITLE
Added Share document library action for OCR. Action can be executed by admins in an event of OCR failure manually on specific documents.

### PR DESCRIPTION
**Pull request by abhinavmishra14**
- Added Share document library action for OCR. Action can be executed by admins in an event of OCR failure manually on specific documents. Action displays for PDF documents only. 

 ocr-share-action

- There were a lot of resource cache warnings whenever we start the containers, i have disabled the resource caching since its test setup.
- Updated readme documentation to provide details about the arguments and also added a missing JAVA_OPTS param in case of EE setup. 
- Allow generation amp modules. Useful if someone wants to install as amp rather than jar.

### CODE REVIEW

File: ocr-share-action/src/main/resources/alfresco/web-extension/messages/ocr-share-action.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+start.ocr.action.success=Started the OCR processing successfully..
```
Comment (aborroy) [inline ocr-share-action/src/main/resources/alfresco/web-extension/messages/ocr-share-action.properties]: ellipsis or single period?

**Review by aborroy — APPROVED** [review]
Minor suggestion for action message

**Comment by sergiobaiao** [comment]
can you update the docker images on docker hub with this version?

**Comment by aborroy** [comment]
@sergiobaiao You just need to apply https://github.com/aborroy/alf-tengine-ocr/tree/master/ocr-share-action addon to your Alfresco Share Docker Image. No requirement to change Repository Docker Image.

**Comment by abhinavmishra14** [comment]
Correct Angel. May be i would update the documentation about “how to
install the add-on share.war or share docker image”. Hopefully that will
clarify some of these doubts

**Pull request merged**